client-java/instrumentation/pom.xml

```diff
@@ -47,6 +47,18 @@
             <artifactId>mockito-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+        </dependency>
+
+
+        <!-- Dependencies for tracking MongoDB queries -->
+        <dependency>
+            <groupId>org.mongodb</groupId>
+            <artifactId>mongodb-driver</artifactId>
+        </dependency>
+
```

Comment:
this is going to be complicated to explain... let's discuss it in a video chat.
Summary:
- we can't have something like this, as all dependencies in `instrumentation` must be _shaded_
- shading bring its own sets of problems... many problems...

I should really put some documentation in this pom file, and issue a WARNING to speak with me before starting to modify it... :)

Reply:
dependecy removed